### ISSUE TITLE
nmsg documentation

### ISSUE THREAD

**Issue by deteque** [farsightsec/nmsg#3]
As new channels have started appearing it's once again becoming obvious that the lack of nmsg documentation is slowing down adoption.  While nmsgtool does produce a splash page of options, it lacks examples showing the correct combination of commands/syntax, the required support libraries, where to obtain those libraries and how to read the nmsg binary files with C and python.

I don't consider "source code" as documentation. :)

In summary, documentation regarding nmsg and nmsgtool would be greatly appreciated.

Andy
